pkg/util/wait/backoff_test.go

```diff
@@ -42,8 +42,8 @@ func makeSpyTimer() SpyTimer {
 	return SpyTimer{history: ptr.To([]time.Duration{}), Timer: timer}
 }
 
-func ms(m time.Duration) time.Duration {
-	return time.Millisecond * m
+func ms(milliseconds int) time.Duration {
+	return time.Duration(milliseconds) * time.Millisecond
```

Comment:
```suggestion
func ms(m int) time.Duration {
	return time.Duration(m) * time.Millisecond
```

I prefer to align the Google Go code style: https://google.github.io/styleguide/go/decisions#receiver-names

Reply:
Ok, I change according to the suggestion.

But `m` is not a receiver name in this case. It's a simple function parameter.